pkg/apis/core/types.go

```diff
@@ -3813,6 +3829,28 @@ type PodSecurityContext struct {
 	// Note that this field cannot be set when spec.os.name is windows.
 	// +optional
 	AppArmorProfile *AppArmorProfile
+	// seLinuxChangePolicy defines how the container's SELinux label is applied to all volumes used by the Pod.
+	// It has no effect on nodes that do not support SELinux or to volumes does not support SELinux.
+	// Valid values are "MountOption" and "Recursive". If not specified, "MountOption" is used.
```

Comment:
>If not specified, "MountOption" is used.
Not yet right? I am not sure if the nuanced way this is being implemented needs to be documented in API docs. But this certainly feels incorrect here technically.